packages/mux-uploader/README.md

```diff
@@ -318,9 +318,11 @@ If an error occurs during the upload, an `uploaderror` event will fire.
-| `upload-in-progress` | `boolean`           | (Read-only) Toggles visual status of progress bar while upload is in progress. Can be targeted with CSS if you want to control styles while in progress i.e. mux-uploader[upload-in-progress].                                                                                        | false       |
-| `upload-error`       | `boolean`           | (Read-only) Toggles visual status of progress bar when upload encounters an error. Can be targeted with CSS if you want to control styles when an error occurs i.e. mux-uploader[upload-error].                                                                                       | false       |
-| `dynamic-chunk-size` | `boolean`           | Toggles uploading with dynamic chunk sizes. Chunk sizes will change with upload speed to attempt to optimize upload.                                                                                                                                                                  | false       |
+| `upload-in-progress` | `boolean`           | (Read-only) Toggles visual status of progress bar while upload is in progress. Can be targeted with CSS if you want to control styles while in progress i.e. `mux-uploader[upload-in-progress]`.                                                                                        | `false`       |
+| `upload-error`       | `boolean`           | (Read-only) Toggles visual status of progress bar when upload encounters an error. Can be targeted with CSS if you want to control styles when an error occurs i.e. `mux-uploader[upload-error]`.                                                                                       | `false`       |
+| `upload-complete`       | `boolean`           | (Read-only) Toggles visual status of progress bar when an upload is completed. Can be targeted with CSS if you want to control styles when the upload completes i.e. `mux-uploader[upload-error]`.                                                                                       | `false`       |
+| `dynamic-chunk-size` | `boolean`           | Toggles uploading with dynamic chunk sizes. Chunk sizes will change with upload speed to attempt to optimize upload.                                                                                                                                                                  | `false`       |
+| `use-large-file-workaround` | `boolean`           | Enables a less memory-efficient way of loading and chunking files for environments that don't reliably handle [`ReadableStream` for large files](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API/Using_readable_streams).      | `false`       |
```

Comment:
Is there a caniuse page that might be helpful to link here?

Reply:
no bc it was a bug 💔 there is a webkit bug link though. Should I add?